common/lib/capa/capa/templates/choicetext.html

```diff
@@ -62,15 +65,17 @@
 
     <div class="indicator-container">
     % if input_type == 'checkbox' or not element_checked:
-        <span class="status ${status.classname}" id="status_${id}"></span>
+        <span class="status ${status.classname}" id="status_${id}">
+            <span class="sr">${status.display_name}</span>
+        </span>
     % endif
     </div>
 
     % if show_correctness == "never" and (value or status not in ['unsubmitted']):
     <div class="capa_alert">${_(submitted_message)}</div>
     %endif
     % if msg:
-    <span class="message">${msg|n}</span>
+    <span class="message" aria-describedby="forinput${choice_id}" aria-live="polite" tabindex="-1">${HTML(msg)}</span>
```

Comment:
We definitely don't want forinput${choice_id} here, as choice_id is defined inside of a loop (see lines 17-18).

@cptvitamin, would it be OK for aria-describedby here to point to the fieldset (line 16)?